charts/spire/charts/spire-server/values.yaml

```diff
@@ -445,13 +445,15 @@ controllerManager:
     - local-path-storage
 
   identities:
+    ## @param controllerManager.identities.defaultSpiffeIDTemplate Spiffe ID template to use for clusterSPIFFEIDs that don't specify one
+    defaultSpiffeIDTemplate: spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}
     clusterSPIFFEIDs:
       # NOTE you can add multiple uniquely named entries to create multiple ClusterSPIFFEID objects. See example below.
       default:
         ## @param controllerManager.identities.clusterSPIFFEIDs.default.enabled Enable this identity for controller manager
         enabled: true
-        ## @param controllerManager.identities.clusterSPIFFEIDs.default.spiffeIDTemplate Spiffe ID template for identities
-        spiffeIDTemplate: spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}
+        ## @param controllerManager.identities.clusterSPIFFEIDs.default.spiffeIDTemplate Set to specify Spiffe ID template for this clusterSPIFFEID
+        spiffeIDTemplate: ""
```

Comment:
This feels a bit like a duplicate from external API point of view. Can't we do the implementation as such it can rely on the original `default.spiffeIDTemplate` field?

Reply:
Interesting. I did it this way as there isn't anything special about the default object. But it is always there... so we could treat it special like that...

Yeah, let me take a stab at implementing that.

Reply:
Yeah, that worked out well. Updated.